src/helpers/UserGen.java

```diff
@@ -240,6 +236,14 @@ public UserGen() {
             gc.set(Calendar.DAY_OF_YEAR, dayOfYear);
             dob = new Date(gc.getTime().getTime());
 
+            Integer age = Period.between(dob.toLocalDate(), LocalDate.now()).getYears();
```

Comment:
Between dob and localdate will be dob and this year won't it? Am I missing something here?